### PR TITLE
Type __subclasses__()

### PR DESCRIPTION
**Pull request by adrian17**
Wanted to get a feel of the Weak stuff.

### CODE REVIEW

**Comment by cthulahoops** [comment]
Cool - I didn't know this existed. I like that you've made this work without a gc whereas cpython needs one.

Looks good to me.

**Comment by adrian17** [comment]
> I like that you've made this work without a gc whereas cpython needs one.

Pretty sure I did implemented this the same way as CPython. I'm assuming additional references in CPython come from some other feature.

File: vm/src/obj/objobject.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ use crate::pyobject::{
 };
 use crate::vm::VirtualMachine;
 
-#[derive(Clone, Debug)]
```
Comment (windelbouwman) [inline vm/src/obj/objobject.rs]: Less is more. I like it!

**Comment by cthulahoops** [comment]
Yeah, I suppose this has to be the same as CPython.

Can CPython ever reference count a class out existence? Do classes have a reference to themselves in their own mro?

**Comment by adrian17** [comment]
> Do classes have a reference to themselves in their own mro?

Yeah, pretty sure that's the case.

**Comment by adrian17** [comment]
anything blocking merging this?

**Pull request merged**